libraries/kubernetes.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2018, ipt switzerland AG.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# author: Cyrill Ruettimann
+
+class Kubernetes < Inspec.resource(1)
+  name 'kubernetes'
+  desc 'Custom resource which abstracts the various kubernetes runtimes like hyperkube'
+
+  def initialize
+    @file = inspec.file('/usr/bin/hyperkube')
+    @ishyperkube = @file.file?
+    Log.debug("The installation is hyperkube=#{@ishyperkube}")
+  end
+
+  def hyperkube
```

Comment:
we should name this `hyperkube?`

Reply:
I think this will then be obsolete when the abstract the decision in kubernetes class - right?

Reply:
Right, I think we can get away without this for now. We only need this if you think there are cases where test output will be different for hyperkube. But we could add this later if really required.